src/main/groovy/pl/allegro/tech/build/axion/release/infrastructure/git/GitRepository.groovy

```diff
@@ -83,12 +87,16 @@ class GitRepository implements ScmRepository {
     }
 
     private void callPush(String remoteName, boolean all) {
-        repository.push(remote: remoteName, all: all)
+        if(pushTagsOnly == false) {
+            repository.push(remote: remoteName, all: all)
+        }
         repository.push(remote: remoteName, tags: true, all: all)
     }
 
     private void callLowLevelPush(ScmIdentity identity, String remoteName, boolean all) {
-        pushCommand(identity, remoteName, all).call()
+        if(pushTagsOnly == false) {
```

Comment:
Indeed. I can only find one instance of '== false' in the codebase, and it's in a test. I'm changing it to the compact syntax to conform.